bundle/print_needed_elf.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+# Prints resolved paths to needed libraries for an ELF executable.
+# ldd also does this, but it segfaults in some odd scenarios so we avoid it.
+import sys
+import os
+import subprocess
+from typing import Any, Iterable, List
+
+
+def eprint(msg: Any):
+    print(msg, file=sys.stderr)
+
+
+def run(args: List[str]) -> str:
+    result = subprocess.run(args, capture_output=True)
+    if result.returncode != 0:
+        eprint(result.stderr)
+        eprint("Command failed with return code {}: {}".format(result.returncode, args))
```

Comment:
if you use non-ancient python
```suggestion
        eprint(f"Command failed with return code {result.returncode}: {args}")
```

Reply:
+1, we use f-string literals in Pwndbg codebase; idk if this is executed from a non-ancient Python tho